charts/fybrik/templates/manager-deployment.yaml

```diff
@@ -78,6 +78,7 @@ spec:
             {{- if and .Values.clusterScoped .Values.manager.prometheus }}
             - "--metrics-bind-addr=127.0.0.1:8080"
             {{- end }}
+            - "--health-probe-addr=127.0.0.1:8088"
```

Comment:
please add to the manager deployment, something like 
```yaml
readinessProbe:
  httpGet:
    path: /readyz
    port: 8088
  failureThreshold: 3
  periodSeconds: 10
  successThreshold: 1
  timeoutSeconds: 1          
  initialDelaySeconds: 0
livenessProbe:
  httpGet:
    path: /healthz
    port: 8088
 failureThreshold: 3
 periodSeconds: 10
 successThreshold: 1
 timeoutSeconds: 1  
```

Reply:
Done